docs/tutorial/3-class-based-views.md

```diff
@@ -28,14 +30,17 @@ We'll start by rewriting the root view as a class based view.  All this involves
             if serializer.is_valid():
                 serializer.save()
                 return Response(serializer.data, status=status.HTTP_201_CREATED)
-            return Response(serializer.errors, status=status.HTTP_400_BAD_REQUEST)
+            else:
+		return Response(serializer.errors, status=status.HTTP_400_BAD_REQUEST)
 
 So far, so good.  It looks pretty similar to the previous case, but we've got better separation between the different HTTP methods.  We'll also need to update the instance view in `views.py`.
 
     class SnippetDetail(APIView):
         """
-        Retrieve, update or delete a snippet instance.
+        Retrieve, update, or delete a snippet instance.
         """
+	queryset = Snippet.objects.all()
```

Comment:
`queryset` is not required here - we're writing the `get_object` logic explicitly.